src/zope/component/tests/test_hookable.py

```diff
@@ -68,62 +68,38 @@ def bar():
     def test_cant_assign_original(self):
         from zope.component.hookable import hookable
         def foo():
-            pass
+            raise NotImplementedError()
```

Comment:
In my opinion, no.

I'll try to explain my thinking about the use of `pass` vs `NotImplementedError` in tests.

`pass` is a legitimate thing to write with a specific meaning: This is a function, it *should* be called, although it doesn't actually do anything. If it's not called, coverage will go down, and *something* changed---there's probably a regression.

In contrast, `raise NotImplementedError` has a different but still specific meaning: this is a callable, and I have to give it a name for reasons out of my control, *but it must not* be called. If it is called, *something* changed---there's probably a regression.

I see those two cases as useful distinctions to make. 

I guess the "must not be called" case could usually be spelled "self.fail", although one would have to capture the correct "self" (or the method "self.fail"). Or we could introduce a specific exception class ("MustNotBeCalledError") to raise instead. Or we could have a pre-defined function that we simply assign to the needed name ("foo") in most cases: `foo = callable_that_is_never_called`)...hey, that has the benefit of being a line shorter!

I wish there were a simple way to spell the `pass` (must be called) case that would result in test failures instead of just coverage decreases.

Thoughts?